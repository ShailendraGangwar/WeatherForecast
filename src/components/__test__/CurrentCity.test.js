import 'react-native';
import React from 'react'
import CurrentCity from '../CurrentCity';

import renderer from 'react-test-renderer';

it('should renders', () => {
  const tree = renderer.create(<CurrentCity />).toJSON();
  expect(tree).toMatchSnapshot();
});