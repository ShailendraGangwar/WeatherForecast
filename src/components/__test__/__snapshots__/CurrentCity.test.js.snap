// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renders 1`] = `
<Text
  style={
    Object {
      "alignItems": "center",
      "color": "red",
      "fontSize": 18,
      "fontWeight": "bold",
      "height": 30,
      "justifyContent": "center",
      "marginTop": 1,
    }
  }
>
   
   
</Text>
`;
